pkgs/development/libraries/libpst/default.nix

```diff
@@ -25,7 +45,7 @@ stdenv.mkDerivation rec {
     homepage = "https://www.five-ten-sg.com/libpst/";
     description = "A library to read PST (MS Outlook Personal Folders) files";
     license = licenses.gpl2;
```

Comment:
Looks like `gpl2Plus`: http://hg.five-ten-sg.com/libpst/file/tip/xml/libpst.in#l287